docs/explanations/policy-groups.md

```diff
@@ -0,0 +1,212 @@
+---
+sidebar_label: Policy Groups
+sidebar_position: 21
+title: Policy Groups
+description: A description of Kubewarden policy groups
+keywords: [ kubewarden, policy groups, clusteradmissionpolicygroup, admissionpolicygroup]
+doc-persona: [kubewarden-operator]
+doc-type: [explanation]
+doc-topic: [explanations, policy-group]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/explanations/policy-groups"/>
+</head>
+
+The policy group feature allows users to create complex policies by combining
+multiple simpler ones. It introduces two new Custom Resource Definitions
+(CRDs):
+
+- `AdmissionPolicyGroup`: For admission policies that apply to specific
+  namespaces. 
+- `ClusterAdmissionPolicyGroup`: For admission policies that apply across the
+  entire cluster.
+
+These policy groups enable users to leverage existing policies, reducing the
+need for custom policy creation and enhancing reusability. By avoiding
+duplication of policy logic, users can simplify management and create custom
+policies with a DSL-like configuration.
+
+The primary goal of policy groups is to facilitate the composite evaluation of
+multiple policies using a boolean expression, allowing for the definition of
+complex logic. However, it is important to note that while ordinary policies
+can include mutation logic to modify resources during admission, policy groups
+are limited to validation only.
+
+Most of the configuration for policy groups is similar to that of ordinary
+policies. The main differences include the addition of the `expression`,
+`message`, and `policies` fields, as well as the declaration of context-aware
+rules in a different location.
+
+## `policies` in a Policy Group
+
+The policies field is a map of ordinary policies (`ClusterAdmissionPolicy` and
+`AdmissionPolicy`) that make up the policy group. These are other Kubewarden
+policies invoked by the policy group to determine whether the resource under
+evaluation is accepted or rejected. The definitions of these policies are a
+simplified version of the ordinary Kubewarden policies, containing only the
+`module`, `settings`, `contextAwareResources`, and `name` of the policy. These
+elements are necessary for the policies to function within a policy group,
+while all other configurations are defined within the policy group itself, as
+    the policy group is considered a single policy. 
```

Comment:
```suggestion
The policies field is a map of ordinary policies (`ClusterAdmissionPolicy` and
`AdmissionPolicy`) that make up the policy group. Kubewarden
policies are called by the policy group, to determine whether the resource under
evaluation is accepted or rejected. The definitions of these policies are a
simplified version of ordinary Kubewarden policies, containing only the
`module`, `settings`, `contextAwareResources`, and `name` of the policy. These
elements are necessary for the policies to function within a policy group.

```

I've taken this bit out 

'while all other configurations are defined within the policy group itself, as
the policy group is considered a single policy.' 

I'm not sure what you are trying to say here. Maybe a little rewording is necessary.?